packages/react-charts/src/echarts/components/Line/examples/next/Charts

```diff
@@ -0,0 +1 @@
+../../../Charts
```

Comment:
Just curious: what's this file doing?

Reply:
This is a soft link for docs. Specifically, so I can show ChartDocs for the "next" line example

FYI, in order to document some of the EChart properties, our docs force us to override each property. Some EChart types are not exported and too complex to recreate. Attempting to duplicate each EChart type would be error-prone -- we end up changing the type in some cases. For documentation purposes, I created ChartDocs, which avoids having to duplicate EChart's complex object types.

Reply:
Removing it locally doesn't seem to change the next line examples from what I can tell 🤔

Reply:
I removed the soft link in favor of importing `propComponents` from relative path -- found the solution here 
https://github.com/patternfly/patternfly-org/pull/3397